answers/Interfaces/98d362bb-a1be-4dcd-84d1-4ccb3c321591/Student.php

```diff
@@ -0,0 +1,24 @@
+<?php
+require_once(__DIR__ . '/Person.php');
+require_once(__DIR__ . '/LifeCycle.php');
+class Student extends Person implements LifeCycle
+{
+    public function species()
+    {
+        return "human";
+    }
+
+    public function stage()
+    {
+        return "adolescent";
+    }
```

Comment:
Can we have a variation on the implementation? Like  in this class `species` can be `mamals` and/or `stage` is `child`?

Reply:
In this lesson,  we only have one set of program shown to the learner with different error location and types this is why in our answer files all the values are similar but in the content the errors and the location of errors which they have to fix are different.